enhancements/ingress/mutable-publishing-scope.md

```diff
@@ -0,0 +1,362 @@
+    reason: ScopeChanged
+    status: "True"
+    type: Progressing
+```
+
+Note that the IngressController remains available, and that the user can revert
+the change by setting `spec.endpointPublishingStrategy.loadBalancer.scope` back
+to its previous value.  That means the user must take one of two actions:
+
+1. Delete the Service referenced in the status condition.
+2. Revert the change to the IngressController.
+
+If the user deletes the Service, the operator recreates it with the desired
+scope.  Alternatively, if the user reverts the change, the `Progressing` status
+condition changes back to `False`, and the state of the IngressController is as
+it was before the user first changed the value of
+`spec.endpointPublishingStrategy.loadBalancer.scope`.
+
+In addition to deleting the Service explicitly, it is possible to annotate the
+IngressController with the newly defined
+`ingress.operator.openshift.io/auto-delete-load-balancer` annotation.  If the
+operator observes that this annotation is set and that its
+`spec.endpointPublishingStrategy.loadBalancer.scope` field has been changed, the
+operator automatically deletes the Service if needed to complete a scope-change
+operation.  This purpose of this annotation is to simplify automation: A tool
+can set the annotation when it creates the IngressController or when it updates
+`spec.endpointPublishingStrategy.loadBalancer.scope` to instruct the operator to
+perform the operation automatically.  This annotation is not intended for
+end-users to use directly.
+
+
+### Validation
+
+This enhancement adds no new API fields or values and requires no new API
+validation.
+
+### User Stories
+
+#### As a cluster administrator, I want my default IngressController to be private (internal)
+
+If the user installs a new cluster without specifying that the cluster is
+"private", the default IngressController is created with "External" scope.  This
+enhancement enables the cluster administrator to change the scope to "Internal"
+as follows:
+
+```shell
+oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"scope":"Internal"}}}}'
+```
+
+The cluster administrator can check the status of the IngressController as
+usual:
+
+```shell
+oc -n openshift-ingress-operator get ingresscontrollers/default -o yaml
+```
+
+The `Progressing` status condition will indicate whether the cluster
+administrator must take further action.  The status condition may indicate that
+the cluster administrator needs to delete the Service, like so:
+
+```shell
+oc -n openshift-ingress delete services/router-default
```

Comment:
This is just stating that if you went from External=>Internal and then issued the delete the operator tracks the the scope you wanted and it will come back as Internal - correct?

Reply:
More or less.  The main point that I'm trying to convey is that the user must explicitly perform the delete operation if one is needed, but to your question, yes, the operator will recreate the service with the scope specified in the ingresscontroller.  Is the current phrasing clear, or does it need some wordsmithing?